setup.py

```diff
@@ -75,6 +80,7 @@
             "oschmod>=0.3.12"
         ],
         "localprocessing": localprocessing_require,
+        "ondemandpreview": ondemandpreview_require,
```

Comment:
I'm wondering if we should put this feature under a very specific extra "ondemandpreview". 
Maybe it's more user friendly to just define a generic extra "jupyter" that allows a user to enable various jupyter notebook related extra/optional features (such as the "ondemand preview" feature). Another feature under this umbrella could be a bounding box selection widget based on ipywidget.

what do you think @m-mohr @clausmichele ?

Reply:
I don't have a strong opinion as I'm not so much into all the details. Is jupyter the right name though? I also felt like "ondemandpreview" is quite lengthy.

I assume the Jupyter rendering via Vue components would stay in core, right?

Reply:
Was done.